cmd2/ansi.py

```diff
@@ -6,10 +6,10 @@
 import functools
 import re
 from enum import Enum
-from typing import Any, IO, List, Union
+from typing import IO, Any, List, Union
```

Comment:
This is the only import I see where it wasn't made alphabetical. Is this the isort version? Does it place all uppercase imports first?

Reply:
Yeah this is how isort does it.  It assumes that anything in all-caps is a constant per Pep8 naming conventions and it puts these first before non-constants.